addons/vehiclelock/stringtable.xml

```diff
@@ -112,4 +112,4 @@
             <Russian>Ключ для открытия большинства машин Гражданских.</Russian>
```

Comment:
My bad, was previewing the diff in collapsed view and all I saw was the English INDEP. You are right, it's CIV here.